hazelcast-jet-core/src/main/java/com/hazelcast/jet/Processors.java

```diff
@@ -823,6 +828,154 @@ public void process(int ordinal, @Nonnull Inbox inbox) {
     }
 
     /**
+     * Convenience for {@link #writeSystemOut(Distributed.Function)} without format.
+     * It will use {@link Object#toString()}.
+     */
+    public static Distributed.Supplier<Processor> writeSystemOut() {
+        return writeSystemOut(null);
+    }
+
+    /**
+     * Returns a supplier of processors, that sends all events to logger on the
+     * INFO level.
+     * <p>
+     * Note, that the event will be logged on the nodes, not on the client.
+     * Useful for testing.
+     *
+     * @param toStringF Function to convert item to String, if {@code null},
+     *                  {@link Object#toString()} is used.
+     */
+    public static Distributed.Supplier<Processor> writeSystemOut(
+            @Nullable Distributed.Function<Object, String> toStringF
+    ) {
+        return () -> new WriteSystemOutP(toStringF);
+    }
+
+    /**
+     * See {@link #peekInput(Distributed.Function, Predicate, ProcessorMetaSupplier)}
+     */
+    public static Distributed.Supplier<Processor> peekInput(Distributed.Supplier<Processor> wrapped) {
+        return peekInput(null, null, wrapped);
+    }
+
+    /**
+     * See {@link #peekInput(Distributed.Function, Predicate, ProcessorMetaSupplier)}
+     */
+    public static Distributed.Supplier<Processor> peekInput(
+            Distributed.Function<Object, String> toStringF, Predicate<Object> shouldLogF,
+            Distributed.Supplier<Processor> wrapped) {
```

Comment:
Here and elsewhere, instead of accepting `null` we should annotate with `@Nonnull` and have our library provide convenient default function factories, like `DistributedFunctions.alwaysTrue`. Where applicable we can add more overloads.